variables/builtin.go

```diff
@@ -22,10 +23,11 @@ type DefaultArgs struct {
 }
 
 // BuiltinArgs returns a scope containing the builtin args.
-func BuiltinArgs(target domain.Target, platr *platutil.Resolver, gitMeta *gitutil.GitMetadata, defaultArgs DefaultArgs, ftrs *features.Features) *Scope {
+func BuiltinArgs(target domain.Target, platr *platutil.Resolver, gitMeta *gitutil.GitMetadata, defaultArgs DefaultArgs, ftrs *features.Features, push bool) *Scope {
 	ret := NewScope()
 	ret.AddInactive(arg.EarthlyTarget, target.StringCanonical())
 	ret.AddInactive(arg.EarthlyTargetProject, target.ProjectCanonical())
+	ret.AddInactive(arg.EarthlyPush, fmt.Sprintf("%t", push))
```

Comment:
We should feature-flag the presence of this builtin arg - mainly because someone could be writing an Earthfile that depends on this and could be sharing with a colleague that has an older Earthly - which would result in different behaviors.

Reply:
Do you think it's sufficient to use the `--wait-block` feature for this, or a separate one?
Technically `wait-block` has already been released, so a user on v0.6.19 could think they are using it, but would need a newer version. However since we say it's experimental, and I print out an extra warning in v0.6.19 saying it's not-fully implemented, I think it's safe-enough to re-use it. Thoughts?

Reply:
Yeah, we can re-use it. We can reuse any feature flag, in general until it has actually been released as part of a `VERSION` number. We don't guarantee forwards-backward compatibility if someone uses feature flags directly - so we're good.